content/en/tutorials/lotus-miner/add-seal-worker.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Add A Seal Worker To Lotus Miner"
+description: "This is a step by step guide on how to setup a lotus seal-worker and connect it to the miner in calibnet."
+lead: "This is a step by step guide on how to setup a lotus seal-worker and connect it to the miner in calibnet. Some of the steps are specific to the hardware and configuration used in this setup and might not be applicable for everyone. Please follow the documentation to set up your miner and use this guide only as a reference point."
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-providers"
+weight: 120
+toc: true
+---
+
+## Setup Details
+
+#### PC1 Worker
+
+- CPU: 2 x AMD EPYC 7F32 8-Core Processor
+- RAM: 1007 GiB
+- GPU: None
+- Process: PC1 Seal Worker
+- OS: Ubuntu
+- Private IP: z.z.z.z
+- Public IP: C.C.C.C
+
+All lotus process will run as a non-root user. Please make sure to open relevant ports in your firewall to allow connections.
+This tutorial is use the miner setup under the How To Run A Miner tutorials.TODO: reflink
+
+## Lotus Worker Setup
+
+### Installation
+
+1. Install Lotus binaries by following the [Linux install guide - build from source]({{<relref "../../lotus/install/ubuntu/#building-from-source" >}}).
+
+```shell
+sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl clang build-essential hwloc libhwloc-dev wget -y && sudo apt upgrade -y
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Choose the default installation option 1) for the above rust installation.
+
+```shell
+wget -c https://golang.org/dl/go1.16.4.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
+
+echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && source ~/.bashrc
+
+git clone https://github.com/filecoin-project/lotus.git
+cd lotus/
+
+git checkout tags/v1.14.4
+
+export CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__"
+export CGO_CFLAGS="-D__BLST_PORTABLE__"
+export FFI_BUILD_FROM_SOURCE=1
+export RUSTFLAGS="-C target-cpu=native -g"
+```
+
+```shell
+make clean calibnet
+
+./lotus --version
+
+```
+
+2. Add swap to the machines as per availability and requirements.
+
+### Configuration
+
+1. Add the below variables to enable logging to a file.
+
+```shell
+export GOLOG_OUTPUT=file >> ~/.bashrc
+export GOLOG_FILE="$HOME/worker.log" >> ~/.bashrc && source ~/.bashrc
+```
+
+2. On the worker machines, create directories to store cache. Make sure these directories are on a fast NVME disk. Otherwise, it will slow down your worker.
+
+```shell
+mkdir ~/parent_cache
+mkdir ~/parameter_cache
+```
+
+### Setup
+
+1. Add the following variables to the ~/.bashrc file and source the file. TODO: Add link to token gen from run a miner tutorial.
```

Comment:
```suggestion
1. Add the following variables to the `~/.bashrc` file and source the file. TODO: Add link to token gen from run a miner tutorial.
```

Surround variables and filenames in code tags.

What is this TODO still doing here?